### ISSUE TITLE
Move binaries from sources of unit test projects in Level1

### ISSUE THREAD

**Issue by k-rus** [Starcounter/Starcounter#386]
Currently, most or some unit test projects build binaries locally inside the project folders, which are under `src`. The binaries are not created in the build output folder, since everything there is packaged into Starcounter installation. Binaries of unit tests are clearly not part of it, but creating them into `src` is also incorrect strategy.

**Comment by dan31** [comment]
Seems that is off the table now due to CMake build, right?

**Comment by k-rus** [comment]
No, cmake is not used for Level1. It is used only for Bluestar in pnext.
